data/transport missions.txt

```diff
@@ -1951,3 +1951,244 @@ ship "Science Drone"
+			choice
+				`	"Alright, I'll get my ship ready."`
+					accept
+				`	"Sorry, but I have better things to be doing."`
+					decline
+	
+	npc
+		government "Pirate"
+		personality nemesis staying harvests plunders
+		system destination
+		fleet "Large Core Pirates" 2
+	npc
+		government "Pirate"
+		personality nemesis staying harvests plunders
+		system "Schedar"
+		fleet "Large Core Pirates"
+		fleet "Small Core Pirates" 3
+	
+	on complete
+		payment
+		payment 10000
+		dialog `Multiple bands of pirate each run by a different warlord are waiting in the spaceport when you land. They take the food from your cargo hold like animals, leaving quite the mess behind. When you return to your ship, you find that the Syndicate has deposited <payment> into your account.`
+
+
+
+mission "Pirate Blockade"
+	minor
+	name `Escape to Freedom`
+	description `Bring <bunks> escaped slaves to freedom from their pirate warlords on <destination>. The woman on <origin> promised that someone would give you <payment> for doing this.`
+	source "Buccaneer Bay"
+	destination "Hippocrates"
+	passengers 19
+	to offer
+		"combat rating" > 500
+	
+	on offer
+		conversation
+			`It is dusk at the <origin> spaceport, and the setting sun casts long shadows across the spaceport deck. You take notice of a slim cloaked figure walking only a few feet behind you who seems to be keeping pace with you.`
+			choice
+				`	(Confront them.)`
+					goto confront
+				`	(Stop walking and see if they stop as well.)`
+			
+			`	You stop be the entrance to a store that claims to have "the best Polecats!" and lean against the building, trying not to draw attention to yourself. The figure keeps walking at the same pace, and as they pass you a soft voice says "Follow me," from under the cloak before the figure bolts off into a dark alleyway next to the building.`
+				goto choice
+			
+			label confront
+			`	You turn around and stare right at the figure, who stops within a foot of you. The shadows cast by the sun make it impossible for you to make out their face under the cloak. "Follow me," a soft voice says from under the cloak before the figure bolts off into a dark alleyway next to the building`
+			
+			label choice
+			choice
+				`	(Follow the figure.)`
+				`	(Return to my ship.)`
+					decline
+			
+			`	You step into the alleyway and find the figure standing under a lamp. The figure removes their cloak, revealing the face of a woman with short dark hair and a distinct jagged scar across her right cheek. She begins whispering to you. "You're not from this world. I can tell. You're too clean and unscarred. Would you be able to help me?"`
+			choice
+				`	"What do you want?"`
+					goto need
+				`	"Why would I help a pirate?"`
+			
+			`	The woman chuckles. "Not everyone who lives on a pirate world is a pirate, you know. Maybe you need to learn what a stereotype is. Regardless..." she trails off.`
```

Comment:
If you want to end that particular statement with an ellipsis, `she` should be capitalized.